charts/nitro/templates/statefulset.yaml

```
@@ -60,8 +60,17 @@ spec:
           image: "{{ .Values.image.repository }}:{{ .Values.image.tag | default .Chart.AppVersion }}"
           imagePullPolicy: {{ .Values.image.pullPolicy }}
           {{- if .Values.diagnosticMode }}
-          command: ["sleep"]
-          args: ["infinity"]
+          command: ["/bin/sh"]
+          args:
+            - -c
+            - |
+              #!/usr/bin/env bash
+              set -eo pipefail
+
+              trap 'exit 0' SIGTERM
```

I got the following error when running this on a test pod `trap: SIGTERM: bad trap` seems like some shells expect the signal without the `SIG` part
```suggestion
              trap 'exit 0' TERM
```